livepatch-calc.c

```diff
@@ -64,11 +70,16 @@ static int livepatch_calc_init(void)
         return ret;
     }
     return 0;
+    #endif
 }
 
 static void livepatch_calc_exit(void)
 {
+    #if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 1, 0)
+    klp_shadow_free_all(SV_COUNTER, NULL);
```

Comment:
Since we didn't use shadow variables, maybe we don't have to use `klp_show_free_all`. 
The reference of shadow variables is [livepatch: introduce shadow variable API](https://lwn.net/Articles/731585/)